python/ray/serve/_private/deployment_state.py

```diff
@@ -321,6 +322,11 @@ def node_ip(self) -> Optional[str]:
         """Returns the node ip of the actor, None if not placed."""
         return self._node_ip
 
+    @property
+    def availability_zone(self) -> Optional[str]:
+        """Returns the availability zone the actor resides in, None if not placed."""
```

Comment:
here None means not placed or we don't have az from ClusterNodeInfoCache?

Reply:
Removed `availability_zone` property since it can be fetched by the controller using cache.